### DISCUSSION TITLE
calc.sum function?

### DISCUSSION THREAD

**Discussion by maribox** [typst/typst#4130] [Q&A] [answered]
Hey!
I just asked Claude Opus to improve my typst code and it came up with the following line:

#let sum = calc.sum(lines.map(line => line.amount * line.price))

Now I found out that calc.sum doesn't exist and I was not able to find an alternative for it, now my question is: why? calc.max/calc.min exist, why not calc.sum?

**Comment by PgBiel** [comment, marked as answer]
You can use `lines.map(line => line.amount * line.price).sum()`.

**Reply by maribox** [reply]
Yeah sorry, I totally forgot to check under "Array" in the documentation. Sorry.
